pywbem_mock/_wbemconnection_mock.py

```diff
@@ -225,31 +208,22 @@ class FakedWBEMConnection(WBEMConnection):
     generate responses in the same way the WBEM server would.
 
     For a description of the operation methods on this class, see
-    :class:`pywbem.WBEMConnection`.
+    :ref:`Faked WBEM operations`.
 
     Each object of this class has its own mock repository. A mock repository
     contains multiple CIM namespaces, and each namespace may contain CIM
```

Comment:
I the first sentence "Each object of this class" should probably change to each :class:`~pywbem.FakedWBEMConnection` object.

Also we can simply bring the second sentence (A mock repository) into the first by changing "A mock repository" to "which"

Reply:
both DONE